### DISCUSSION TITLE
Use JAEGER_TAGS to inject tags at the collector level

### DISCUSSION THREAD

**Discussion by zvazquez** [jaegertracing/jaeger#3272] [General]
Hi,

I am looking for the documentation on how to use JAEGER_TAGS while using the Kubernetes Jaeger Operator to inject tags and environment metadata into each span collected by the Operator.

If somebody can point me to the documentation or share with me the details on how to configure the Jaeger custom resource to indicate the operator to use JAEGER_TAGS, looks like there is some support since seems some development was done [here]([url](https://github.com/jaegertracing/jaeger/issues/1844))

Thanks,

Zareh

**Comment by joe-elliott** [comment]
There is a `--collector.tags` param for the collector that allows you to add tags to all spans passing through a collector:

https://www.jaegertracing.io/docs/latest/cli/
